testing/src/main/java/org/motechproject/nms/testing/web/TestingController.java

```diff
@@ -29,4 +33,10 @@ public String sendEvent(@RequestParam(required = true) String subject) {
         return String.format("Sent MOTECH event: %s with empty params: %s", subject, eventParams.toString());
     }
 
+    @RequestMapping(value = "/sendSMS200", method = RequestMethod.POST, headers = { "Content-type=application/json" })
+    @ResponseStatus(HttpStatus.ACCEPTED)
```

Comment:
@koshalt this method return response status code as 202 (Accepted) and response entity as "OK".